lib/galaxy/webapps/galaxy/buildapp.py

```diff
@@ -128,6 +129,9 @@ def paste_app_factory( global_conf, **kwargs ):
 
     register_postfork_function(postfork_setup)
 
+    for th in threading.enumerate():
+        if th.is_alive():
+            log.debug("Prior to webapp return, Galaxy thread %s is alive.", th)
```

Comment:
Yeah, that's what I figured out.  The problem was conflicting environments in that I was using a system uwsgi but the .venv from Galaxy, so it wasn't being found.